### PR TITLE
String tiers django-based views

### PR DESCRIPTION
**Pull request by phlax**
Implementation of the string tiers project views using django/jinja

### CODE REVIEW

**Comment by mathjazz** [comment]
Nice work!

On the `/projects/PROJECT/tags/` dashboard I see 2 things missing:
1. Priority needs to be represented with stars.
1. Tags entry flickers on hover. You can prevent it by adding the following rule to the `table.css`:
```css
.table th.tag {
  width: 330px;
}
```

On the `/projects/PROJECT/tags/TAG/` dashboard everything looks fine in the tab, but the heading needs to follow the [spec](https://user-images.githubusercontent.com/626716/35684935-a14da4ae-0768-11e8-9c25-0c4c18424dc7.png) more closely, for example:
* change the `<title>` tag: Project Tag
* change dashboard title: Project Tag
* add Tag priority to the heading
* reflect only the tag in stats

**Comment by mathjazz** [comment]
Really nice work!

Two last things I have are:

1. In `/projects/PROJECT/tags/TAG/` "PRIORITY" in the heading should ready "PROJECT PRIORITY".
1. In `/projects/PROJECT/tags/` it would be nice to add a search field, same as style as we use for searching teams, projects and resources in other dashboards. It not an absolute must, but since the UI is so similar to other dashboards, the user kind of expects it to be there.

**Comment by phlax** [comment]
> it would be nice to add a search field

i thought about this, but as the expected number of tags is likely to be low initially, i thought it would be kinda superfluous

File: pontoon/tags/urls.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import views
+
+
+urlpatterns = [
+    # Project tag page
+    url(r'^projects/(?P<project>[\w-]+)/tags/(?P<tag>[\w-]+)/$',
+        views.ProjectTagView.as_view(),
+        name='pontoon.tags.project.tag'),
+
+    # AJAX view: Project tag
```
Comment (mathjazz) [inline pontoon/tags/urls.py]: Nit: `# AJAX view: Project tag teams`

File: pontoon/tags/urls.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    url(r'^projects/(?P<project>[\w-]+)/ajax/tags/(?P<tag>[\w-]+)/$',
+        views.ProjectTagView.as_view(),
+        name='pontoon.projects.ajax.tags'),
```
Comment (mathjazz) [inline pontoon/tags/urls.py]: Nit: Looking at the `localization` module, `pontoon.tags.ajax.teams` would be more consistent.

File: pontoon/tags/templates/tags/includes/tags.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<div id="info-wrapper">
```
Comment (mathjazz) [inline pontoon/tags/templates/tags/includes/tags.html]: You can remove this wrapper.

File: pontoon/tags/templates/tags/includes/tags.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+{% import 'tags/widgets/tag_list.html' as TagList %}
+
+<div class="resources">
```
Comment (mathjazz) [inline pontoon/tags/templates/tags/includes/tags.html]: `class="tags"`

File: pontoon/tags/templates/tags/includes/tags.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  {% set latest_activity = tag.latest_activity %}
+  {% set chart = tag.chart %}
+  {{ TagList.item(tag, main_link, chart_link, latest_activity, chart) }}
+  {% endfor %}
+
+  {{ TagList.footer() }}
+</div>
+
+
+</div>
```
Comment (mathjazz) [inline pontoon/tags/templates/tags/includes/tags.html]: This file should go into `projects/templates/projects/includes`. Each file in that folder represents a tab on the project dashboard.

File: pontoon/tags/templates/tags/includes/tag.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{% import 'widgets/latest_activity.html' as LatestActivity %}
```
Comment (mathjazz) [inline pontoon/tags/templates/tags/includes/tag.html]: As far as I can see this file is never used and so is the imported `tag_locales.html` file.

**Pull request merged**